docs/source/processes/index.rst

```diff
@@ -4,21 +4,17 @@
 Processes
 =========
 
-PAVICS offers a fair number of processes that were developed by different groups, but here instead of arranging processes by their source, we split them according to their purpose.
+PAVICS offers a growing list of computational services through the Web Processing Services (WPS) standard. Each individual service is hosted by a thematic server. For example, the :doc:`Finch server <finch:index>` specializes in climate indicators, while the :doc:`Raven server <raven:index>` focuses on hydrological modeling and time-series analysis. A typical server will thus host a few dozens or more individual processes that can be combined into workflows. Indeed, each service typically operates on netCDF inputs and also yields netCDF outputs, making it possible to chain together multiple processes.
```

Comment:
This works perfectly locally. Awesome.

Reply:
Just checked out https://pavics-sdi.readthedocs.io/en/latest/processes/index.html, linking works and I was also able to seach for "cdd" from there and it returns the result from project finch since it's an xclim indicator.

So this cross project search works because of intersphinx or RTD subproject?

Reply:
Subprojects. 
Intersphinx only allows us to create links inside the docs to other objects. It doesn't affect search. Next step is to build the docs in RTD for all the other subcomponents.